build.gradle

```diff
@@ -51,6 +51,29 @@ project(":desktop") {
 
 project(":core") {
     apply plugin: "java"
+    apply plugin: 'pmd'
+    apply plugin: 'findbugs'
+
+    findbugs{
+        ignoreFailures=true
```

Comment:
This should be false. Why is this true? Travis should fail if there is rule validation